sources/packages/web/src/services/ApplicationOfferingChangeRequestService.ts

```diff
@@ -176,7 +191,9 @@ export class ApplicationOfferingChangeRequestService {
    */
   async updateApplicationOfferingChangeRequestStatus(
     applicationOfferingChangeRequestId: number,
-    payload: StudentApplicationOfferingChangeRequestAPIInDTO,
+    payload:
```

Comment:
Do we need to update this?

Approval was not part of story right?

Reply:
I mistakenly started working on the model dialog and added the dialog and this. Then I realized that it wasn't a part of this story and removed the dialog but forgot to remove this. Removed it now.